apiserver/admin_test.go

```diff
@@ -774,10 +784,13 @@ func (s *loginSuite) TestOtherModelWhenNotController(c *gc.C) {
 
 	modelState := s.Factory.MakeModel(c, nil)
```

Comment:
Please add a TODO to the MakeModel method itself in factory to return the model and not the state. A follow up PR will need to then implement that refactoring to remove state from the equation.